### DISCUSSION TITLE
BeeRef 0.3.3 (2024-05-05)

### DISCUSSION THREAD

**Discussion by rbreu** [rbreu/beeref#108] [Anncouncements]
### Added

* Moving the window from within BeeRef now changes to a diffent cursor from the default arrow cursor.
* Added a color sampler which can copy colors from images to the clipboard in hex format (Images -> Sample Color)
* Added notification when attempting to paste from an empty or unusable clipboard
* Added panning via scrollwheel:
** Scroll wheel + Shift + Ctrl: pan vertically
** Scroll wheel + Shift: pan horizontally
* Make mouse and mouse wheel controls configurable (Settings -> Keyboard & Mouse)

### Fixed

* Fixed a crash when pressing the keyboard shortcut for New Scene while in the process of doing a rubberband selection.
* The checkmark of the menu entry Images -> Grayscale is now updating correctly depending on the selected images.
* Removed black line under marching ants outline of crop mode, which would scale with the image and get potentially very thick.
* Fixed a crash when importing images with unsupported exif orientation info 
* Fixed threading issue when importing images (causing potential hangs/weird behaviour)
* Fixed an intermittent crash when invoking New Scene
* Fixed bee files hanging on to disk space of deleted images (issue #99)
* Fixed Drag @ Drop from pinterest feed (by Randommist)
* Fixed pasted items being inserted behind existing items

<hr /><em>This discussion was created from the release <a href='https://github.com/rbreu/beeref/releases/tag/v0.3.3'>BeeRef 0.3.3 (2024-05-05)</a>.</em>

**Comment by senntore** [comment]
Amazing work on the update!!
Just wanted to let you know that when I try to insert a jpg image of size 26.2 MB in an empty scene, I get an error.
Here is the screenshot:
![image](https://github.com/rbreu/beeref/assets/58325574/5ca3cd59-9e29-41ef-8306-9de12cb4e159)

Let me know if I should create an issue for it.

**Reply by rbreu** [reply]
Huh. Qt says about the allocation limit: "It is normally not needed to change it. The default limit is large enough for all commonly used image sizes." 😂

You could try starting from command line like this: `QT_IMAGEIO_MAXALLOC=500 BeeRef-0.3.3.appimage` (Make sure the appimage has executable rights.)

I can make this a configurable setting in the future.

**Reply by senntore** [reply]
Thanks! It works when I set the Max Allocation to 585 & more for image size of 26.2 MB.

**Comment by CDandy** [comment]
Simple and to the point, great work ! I'll try to allocate some time to work on issues and improvements :)

**Comment by yuk159** [comment]
Thanks a lot for the update and the Appimage for Linux, BeeRef is really a great tool.

**Comment by Coltonen** [comment]
Thank you so much for all your hard work! Many details have been carefully considered, and it is immediately obvious that a great deal of effort has gone into this project.

**Comment by sbrl** [comment]
Thanks so much for this release! It fixes a bug for me where I couldn't save anything without crashing ❤️

**Comment by watercoloreater** [comment]
Thanks! ❤️❤️❤️

**Comment by louder-thanbombs** [comment]
Is there anyway to change the opacity of the entire program? I can only figure out how to change the opacity of the picture and I was hping to use the app to move my traditional drawings into electronic
